### PR TITLE
Update amdgpu firmwares to the latest

### PR DESCRIPTION
**Pull request by juikim**
Note the next two commits were support for RDNA2 (Sienna Cichlid) and Cezanne APU (Green Sardine):

https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=54c797a694d3830ef93f349ec25e1d96d4b88102
https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=f7915a0c29fee27a310cebd7155b9e3a6eb71a1d

Since we don't support either of them yet, these two commits were intentionally left out.

### CODE REVIEW

**Comment by evadot** [comment]
LGTM

**Review by zeising — CHANGES_REQUESTED** [review]
Small nit, see inline below.
Also, any chance you can squash the two commits hooking up the new firmware into one commit?

File: amdgpukmsfw/Makefile [hunk 1]
```diff
@@ -119,9 +119,9 @@ SUBDIR=									\
 	navi14_me							\
 	navi14_me_wks							\
 	navi14_mec							\
-	navi14_mec_wks							\
 	navi14_mec2							\
 	navi14_mec2_wks							\
+	navi14_mec_wks							\
```
[the remark this answers is not in the document]
Comment (juikim) [reply amdgpukmsfw/Makefile:124]: I just sorted it with `LANG=C`.  In fact, it was the only file in wrong order.

[the remark this answers is not in the document]
Comment (juikim) [reply amdgpukmsfw/Makefile:124]: > Also, any chance you can squash the two commits hooking up the new firmware into one commit?

Done.

[the remark this answers is not in the document]
Comment (zeising) [reply amdgpukmsfw/Makefile:124]: Can you please adjust this so that firmware that goes for the same card is in the same place in the makefile.

**Review by zeising — CHANGES_REQUESTED** [review]
History still looks quite strange, it is now 6 commits (one is a merge commit) with three commits related to hooking up the new firmware.

File: amdgpukmsfw/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (zeising) [reply amdgpukmsfw/Makefile:124]: Never mind, I see now that this is actually more correct, looking at the rest of the file.  Sorry about that.

**Pull request closed without merging**